codalab/worker/bundle_manager.py

```diff
@@ -250,15 +250,14 @@ def _restage_stuck_starting_bundles(self, workers):
 
     def _fail_stuck_running_bundles(self, workers):
         """
-        Fails bundles that got stuck in the RUNNING state.
+        Make bundles that got stuck in the RUNNING state into WORKER_OFFLINE state.
         """
         for bundle in self._model.batch_get_bundles(state=State.RUNNING, bundle_type='run'):
             if (not workers.is_running(bundle.uuid) or  # Dead worker.
-                time.time() - bundle.metadata.last_updated > 60 * 60):  # Shouldn't really happen, but let's be safe.
-                failure_message = 'Worker died'
+                time.time() - bundle.metadata.last_updated > 1 * 30):  # Shouldn't really happen, but let's be safe.
```

Comment:
Define a constant for `30` and use that in both places rather than having 2 bare constants that need to be kept in sync.